L1Trigger/L1THGCal/interface/backend/HGCalSortingTruncationImpl_SA.h

```diff
@@ -0,0 +1,29 @@
+#ifndef __L1Trigger_L1THGCal_HGCalSortingTruncationImplSA_h__
+#define __L1Trigger_L1THGCal_HGCalSortingTruncationImplSA_h__
+
+#include "L1Trigger/L1THGCal/interface/backend/HGCalCluster_SA.h"
+#include "L1Trigger/L1THGCal/interface/backend/HGCalSeed_SA.h"
+#include "L1Trigger/L1THGCal/interface/backend/HGCalMulticluster_SA.h"
+#include "L1Trigger/L1THGCal/interface/backend/HGCalSortingTruncationConfig_SA.h"
+
+#include <string>
+#include <map>
+#include <vector>
+#include <utility>
+#include <memory>
+
+class HGCalSortingTruncationImplSA {
+public:
+  HGCalSortingTruncationImplSA();
+  ~HGCalSortingTruncationImplSA() {}
+
+  void runAlgorithm() const;
```

Comment:
This function doesn't seem to be defined or used

Reply:
Removed